buildenv/docker/mkdocker.sh

```diff
@@ -33,16 +33,17 @@ usage() {
   echo "  --arch=...     specify the processor architecture (default: host architecture)"
   echo "  --build        build the docker image (overrides '--print')"
   echo "  --cuda         include CUDA header files"
+  echo "  --freemarker   include freemarker.jar"
```

Comment:
The options were listed in alphabetical order, and the variables and cases were ordered likewise.

Reply:
Moved

Reply:
I wasn't going to insist, but thanks.